src/content/get-started/fwe/local-caching.md

```diff
@@ -3,8 +3,10 @@ title: Local caching
 description: Learn how to persist data locally.
 prev:
   title: Networking and data
+  path: /get-started/fwe/networking
 next:
-    title: JSON serialization
+  title: Learn more
+  path: /get-started/learn-more
```

Comment:
This might not be the page we want to link from, but we need to link to something or the page layout near the title gets wonky